README.md

```diff
@@ -47,21 +45,22 @@
     [Windows](https://github.com/aws/q-command-line-discussions/discussions/15)
   - Or [use it on Windows with WSL](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-installing.html#command-line-installing-windows)
 - **Remote machines**
-  - [Autocomplete in SSH](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-autocomplete-ssh.html)
+  - [Chat in SSH](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-chat-ssh.html)
```

Comment:
Both before and after are redirecting to `https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/what-is.html`